patch.yml

```diff
@@ -54,3 +102,5 @@
       db_config_type: RAC
     when: hostvars[groups['dbasm'].0]['cluster_name']|default('', true)|length > 0
   tags: rac-dbpatch
+
```

Comment:
Not sure if you intended to add extra blank lines

Reply:
Ack, Will remove